src/core/webview/ClineProvider.ts

```diff
@@ -320,7 +326,7 @@ export class ClineProvider implements vscode.WebviewViewProvider {
 			task,
 			images,
 			undefined,
-			experimentalDiffStrategy,
+			Experiments.isEnabled(experiments, EXPERIMENT_IDS.DIFF_STRATEGY),
```

Comment:
Do we still need to pass this into Cline? Seems like it would be more consistent to pass experiments